app/models/notification.rb

```diff
@@ -0,0 +1,6 @@
+class Notification < ApplicationRecord
+  belongs_to :sender, class_name: 'User'
+  belongs_to :recipient, class_name: 'User'
```

Comment:
@FaxriddinMaxmadiyorov double quote

Reply:
@XaoGao why we should use double quotes? as far as i know, double quotes are used while interpolating.